<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageExportOptionsWidget</class>
 <widget class="QWidget" name="ImageExportOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>454</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="m_pageLabel">
       <property name="text">
        <string>Page:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="pageCombo"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Size in Pixels</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Image width:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="pxWidth"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Image height:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="pxHeight"/>
      </item>
      <item row="0" column="2" rowspan="2">
       <widget class="KoAspectButton" name="pxAspect" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Size in Units</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Image width:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KoUnitDoubleSpinBox" name="unitWidth"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Image height:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KoUnitDoubleSpinBox" name="unitHeight"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Resolution:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="dpi"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Unit:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="unit"/>
      </item>
      <item row="0" column="2" rowspan="2">
       <widget class="KoAspectButton" name="unitAspect" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Background Color</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="labelColor">
        <property name="text">
         <string>Color</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KColorButton" name="backColor"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelOpacity">
        <property name="text">
         <string>Opacity</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KoSliderCombo" name="opacity"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>12</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KoUnitDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header location="global">KoUnitDoubleSpinBox.h</header>
  </customwidget>
  <customwidget>
   <class>KoAspectButton</class>
   <extends>QWidget</extends>
   <header location="global">KoAspectButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KoSliderCombo</class>
   <extends>QComboBox</extends>
   <header location="global">KoSliderCombo.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
